packages/twitter-verifier-circuits/twitter.circom

```diff
@@ -54,6 +54,7 @@ template TwitterVerifier(max_header_bytes, max_body_bytes, n, k, pack_size, expo
         signal (from_regex_out, from_regex_reveal[max_header_bytes]) <== FromRegex(max_header_bytes)(in_padded);
         log(from_regex_out);
         from_regex_out === 1;
+        in_padded[email_from_idx+max_email_from_len] === 0; // adversary can't use email len > max allowed
```

Comment:
What? Does this array out of bounds indexing work in circom lol

Reply:
oops, yes you are right. Have added correct solution now.